mne/decoding/receptive_field.py

```diff
@@ -337,6 +323,17 @@ def _times_to_delays(tmin, tmax, sfreq):
     return delays
 
 
+def _delays_to_slice(delays):
+    """Find the slice to be taken in order to remove missing values."""
+    # Negative values == cut off rows at the beginning
+    min_delay = np.clip(delays.min(), None, 0)
+    min_delay = None if min_delay >= 0 else -1 * min_delay
+    # Positive values == cut off rows at the end
+    max_delay = np.clip(delays.max(), 0, None)
+    max_delay = None if max_delay <= 0 else -1 * max_delay
+    return slice(min_delay, max_delay)
```

Comment:
does the code require us to force `tmin < 0` and `tmax > 0` as you are doing here?